give.php

```diff
@@ -409,6 +419,10 @@ public function init() {
 			$this->session_db             = new Give_DB_Sessions();
 			$this->session                = Give_Session::get_instance();
 
+			// Load routes.
+			$this->singletonsCache[ FormRoute::class ] = new FormRoute();
+			$this->singletonsCache[ FormRoute::class ]->init( new FormRouteController() );
+
```

Comment:
@JasonTheAdams it is really important to load the route on the plugin load otherwise WordPress will not able to parse URL as we want. To remove it from here, I have to register and action.

updated: https://github.com/impress-org/givewp/compare/852f7719d61d...bf22b45d7013

Reply:
Fixed: https://github.com/impress-org/givewp/compare/bf22b45d7013...89c0fdee6eae